build/nv.d3.js

```diff
@@ -12324,7 +12324,10 @@ nv.models.stackedArea = function() {
                 .width(availableWidth)
                 .height(availableHeight)
                 .x(getX)
-                .y(function(d) { return d.display.y + d.display.y0 })
+                .y(function(d) {
+                  if( typeof d.display === 'undefined' ) {}
+                  else { return d.display.y + d.display.y0 }
```

Comment:
That's an odd way to do it... should probably be this: 

    if (d.display !== undefined) { 
        return d.display.y + d.display.y0;
    }